.github/workflows/automated-zip-to-pr.yml

```diff
@@ -0,0 +1,124 @@
+name: Automated Zip to PR
+
+on:
+  issues:
+    types: [ "opened", "edited" ]
+
+  issue_comment:
+    type: [ "created", "edited" ]
+
+jobs:
+
+ extrakt:
+
+    # We don't do anything on pull requests, which are also considered issues
+    if: ${{ !github.event.issue.pull_request }}
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+      # Only do these if the target is the ISSUE not a COMMENT on the issue
+      - name: Extract Zip URL (body)
+        uses: actions-ecosystem/action-regex-match@v2
+        if: ${{ !github.event.comment.body }}
+        id: match-url
+        with:
+          text: ${{ github.event.issue.body }}
+          regex: '\((.*?\.zip)\)'
+      
+      - name: Extract Zip Filename (body)
+        uses: actions-ecosystem/action-regex-match@v2
+        if: ${{ !github.event.comment.body }}
+        id: match-filename
+        with:
+          text: ${{ github.event.issue.body }}
+          regex: '\(.*\/(.*?\.zip)\)'
+
+      # We do this one ALWAYS so we can compare the issue zip name to the comment zip name
+      - name: Extract Theme Name (body)
+        uses: actions-ecosystem/action-regex-match@v2
+        id: match-theme-name
+        with:
+          text: ${{ github.event.issue.body }}
+          regex: '\(.*\/(.*?)\.zip\)'
+
+      - name: Define Variables (body)
+        if: ${{ !github.event.comment.body }}
+        run: |
+          echo "ZIP_URL=${{ steps.match-url.outputs.group1 }}" >> $GITHUB_ENV
+          echo "ZIP_FILENAME=${{ steps.match-filename.outputs.group1 }}" >> $GITHUB_ENV
+          echo "THEME_NAME=${{ steps.match-theme-name.outputs.group1 }}" >> $GITHUB_ENV
+          echo "BRANCH_NAME=auto/${{ steps.match-theme-name.outputs.group1 }}" >> $GITHUB_ENV
+
+      # Only do these if the target is a COMMENT on the issue
+      - name: Extract Zip URL (comment)
+        uses: actions-ecosystem/action-regex-match@v2
+        if: ${{ github.event.comment.body }}
+        id: match-url-comment
+        with:
+          text: ${{ github.event.comment.body }}
+          regex: '\((.*?\.zip)\)'
+      
+      - name: Extract Zip Filename (comment)
+        uses: actions-ecosystem/action-regex-match@v2
+        if: ${{ github.event.comment.body }}
+        id: match-filename-comment
+        with:
+          text: ${{ github.event.comment.body }}
+          regex: '\(.*\/(.*?\.zip)\)'
+
+      - name: Extract Theme Name (comment)
+        uses: actions-ecosystem/action-regex-match@v2
+        if: ${{ github.event.comment.body }}
+        id: match-theme-name-comment
+        with:
+          text: ${{ github.event.comment.body }}
+          regex: '\(.*\/(.*?)\.zip\)'
```

Comment:
It triggers for any comment made on any issue. It should also be restricted only to issues that have special label.